src/Avalonia.Base/Input/InputExtensions.cs

```diff
@@ -52,26 +61,26 @@ public static IEnumerable<IInputElement> GetInputElementsAt(this IInputElement e
         /// A filter predicate. If the predicate returns false then the visual and all its
         /// children will be excluded from the results.
         /// </param>
+        /// <param name="enabledElementsOnly">Whether to only return elements for which <see cref="IInputElement.IsHitTestVisible"/> is true.</param>
         /// <returns>The topmost <see cref="IInputElement"/> at the specified position.</returns>
         public static IInputElement? InputHitTest(
             this IInputElement element,
             Point p,
-            Func<Visual, bool> filter)
+            Func<Visual, bool> filter,
+            bool enabledElementsOnly = true)
```

Comment:
It certainly should be, but it has been hardcoded in this class the whole time. It would be a breaking change to now expect it to be part of the `filter` parameter of the original method. So I added this overload.